tests/test_utils/test_env.py

```diff
@@ -26,8 +26,9 @@ def test_collect_env():
         for key in ['CUDA_HOME', 'NVCC']:
             assert key in env_info
 
-    if sys.platform != 'win32':
-        assert 'GCC' in env_info
+    assert 'GCC' in env_info
```

Comment:
We can add `GCC` in `expected_keys`.

Reply:
Not sure what it means. Is it only related to unitest?

Reply:
Yet, this line can be removed and add `GCC` to `expected_keys` in line 17.